samples/go/noms-merge/main.go

```diff
@@ -91,6 +98,60 @@ func nomsMerge() error {
 	}))
 }
 
+func resolve(aChange, bChange types.ValueChanged, a, b types.Value, path types.Path) (change types.ValueChanged, merged types.Value, ok bool) {
```

Comment:
@aboodman, @rafael-atticlabs 
Are we really planning to keep this UX around? My understanding was that it was mostly for a demo, so I'm kinda loathe to refactor and add tests for it if we really don't care.